core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java

```diff
@@ -660,6 +660,14 @@ public TagVFilter build() {
             "The tagk cannot be null or empty");
       }
       
+      // V2 librange conversion
+      if (type.equals(TagVLiteralOrFilter.FILTER_NAME)) {
```

Comment:
Doesn't Olympus actually set the `type` to `librange`? I thought it converted properly.

Reply:
Olympus does set it to librange, but the API that's being used by the touchstone/deploy builds doesn't convert those.